src/app/shared/user-navbar/style.scss

```diff
@@ -0,0 +1,86 @@
+@use '@angular/material' as mat;
+@use '../../styles/palettes' as palette;
+
+.nav-bar {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  top: 0;
+  width: 100%;
+  height: 10vh;
+  position: fixed;
+  background-color: mat.get-color-from-palette(palette.$gray-palette, 100);
+  transition-duration: 300ms;
+  transform: translateY(0);
+  &--visi {
+    transform: translateY(-10vh);
+    transition-duration: 300ms;
+  }
+  z-index: 10;
```

Comment:
`visi` to skrót od visible ? Może lepiej pozostać przy pełnych nazwach

Reply:
done 
🧐